### PR TITLE
Optimize dark theme

### PR DESCRIPTION
**Pull request by Edsuns**
### The original border was too bright and unattractive on dark theme. I made some changes and now it looks much better.

Note: I also changed the `footer_fixed` configuration, which probably shouldn't be merged.

### CODE REVIEW

File: _sass/_variables.scss [hunk 1]
```diff
@@ -36,3 +36,4 @@ $black-color: #000000 !default;
 $code-bg-color-light:     rgba($purple-color, 0.05);
 $code-bg-color-dark:      #2c3237 !default;
 
+$dark-bg-color: #1C1C1D;
```
Comment (alshedivat) [inline _sass/_variables.scss:39]: minor: could you please indent this the same way as `$code-bg-color-*` variables and add a new line at the end of the file?

File: _config.yml [hunk 2]
```diff
@@ -31,7 +31,7 @@ impressum_path:  # set to path to include impressum link in the footer, use the
 # -----------------------------------------------------------------------------
 
 navbar_fixed: true
-footer_fixed: true
+footer_fixed: false
```
Comment (alshedivat) [inline _config.yml:34]: please revert this change

**Review by alshedivat — CHANGES_REQUESTED** [review]
looks great, thank you for the contribution!

happy to merge if you could make a couple of small changes requested below. thanks.

File: _sass/_variables.scss (hunk 1 above)
Comment (Edsuns) [reply _sass/_variables.scss:39]: OK. The variable `$dark-bg-color` is supposed to be named with `$grey-color-dark`. I will make changes later.

**Review by alshedivat — APPROVED** [review]
